SPECS/llvm/llvm.spec

```diff
@@ -42,6 +48,8 @@ cmake -G Ninja                              \
       -DCMAKE_BUILD_TYPE=Release            \
       -DLLVM_PARALLEL_LINK_JOBS=1           \
       -DLLVM_BUILD_LLVM_DYLIB=ON            \
+      -DLLVM_INCLUDE_TESTS=ON               \
```

Comment:
What do you think about surrounding these 2 arguments with `if %{with_check}` as well to not increase the build time when we're only building packages?

Reply:
From my local build, adding these did not change the build time significantly